lib/helpers/features.js

```diff
@@ -46,10 +46,10 @@ const DRAFTS = new Map(Object.entries({
     version: [2, 'draft-02'],
   },
   jwtIntrospection: {
-    name: 'JWT Response for OAuth Token Introspection - draft 08',
+    name: 'JWT Response for OAuth Token Introspection - draft 09',
     type: 'IETF OAuth Working Group draft',
-    url: 'https://tools.ietf.org/html/draft-ietf-oauth-jwt-introspection-response-08',
-    version: [8, 'draft-08'],
+    url: 'https://tools.ietf.org/html/draft-ietf-oauth-jwt-introspection-response-09',
+    version: [9, 'draft-09'],
```

Comment:
Remove the 9 please, just leave the string in.

Reply:
Resolved as amendment to the original commit.